ionic_user_interface/src/components/UserRouteList.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+  <ion-list class="ion-no-margin ion-no-padding">
+    <ion-card
+      v-for="(route, index) of routes"
+      :key="index"
+      class="ion-text-left route-card"
+      @click="() => handleRouteCardClick(route.username, route.createdAt)"
+    >
+      <ion-card-header>
+        <ion-card-title>{{ route.routeName }}</ion-card-title>
+        <VoteButton
+          class="vote-button"
+          :username="route.username"
+          :createdAt="route.createdAt"
+          v-model:voteCount="route.voteCount"
+          v-model:hasVoted="route.hasVoted"
+        ></VoteButton>
+      </ion-card-header>
+      <ion-card-content>
+        <b>Gym:&nbsp;</b>
+        <router-link
+          :to="'/gym/' + route.gymLocation + '/' + route.gymName"
+          @click.capture.stop="() => undefined"
+        >
+          {{ route.gymName }}
+        </router-link>
+        <br />
+        <b>Grade:</b>
+        V{{ route.publicGrade }}
+        <br />
+        <b>Created:</b>
+        {{ route.createdAt.split('T')[0] }}
+      </ion-card-content>
+    </ion-card>
+    <ion-card v-if="routes.length === 0" class="ion-text-center route-card">
```

Comment:
Same comment as in GymRouteList.vue, need to remove the `:hover` attribute, if not users will be confused as to why it's not clickable